### ISSUE TITLE
add support for activity_copy_limit on follow_many

### ISSUE THREAD

**Issue by tbarbugli** [GetStream/stream-python#40]
```
follows = [
  {'source': 'group:1', 'target': 'user:1'}, 
  {'source': 'group:1', 'target': 'user:2'},
 ]

stream_client.follow_many(follows,activity_copy_limit=10)
```

**Comment by tbarbugli** [comment]
https://github.com/GetStream/stream-django/issues/41

**Comment by dwightgunning** [comment]
@pterk - it looks like this was actually fixed quite a while back in 126d6aaee. Can you confirm and close if so?

**Comment by pterk** [comment]
@dwightgunning can confirm
